src/main/java/uk/gov/hmcts/reform/jps/domain/SittingRecord.java

```diff
@@ -100,22 +101,17 @@ public LocalDateTime getChangeByDateTime() {
     }
 
     public StatusHistory getFirstStatusHistory() {
-        List<StatusHistory> statusHistoriesCopy = statusHistories.stream()
-            .sorted(Comparator.comparingLong(StatusHistory::getId))
-            .toList();
-        Optional<StatusHistory> optStatHistory = statusHistoriesCopy.stream().findFirst();
+        Collections.sort(statusHistories, Comparator.comparing(StatusHistory::getId));
```

Comment:
Could replace with - statusHistories.stream().min(Comparator.comparingLong(StatusHistory::getId)).orElse(null);

Reply:
Had something similar in another branch. Cool